JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java

```diff
@@ -35,7 +35,7 @@ public static void main(String[] args) {
 				} else if (d.equalsIgnoreCase("q")) {
 					return;
 				} else {
-					console.printf("Sorry, '%s' is not a valid command.\n", d);
+					System.err.println("Sorry, '%s' is not a valid command.\n", d);
```

Comment:
![image](https://user-images.githubusercontent.com/43349586/68947099-4f5e1280-07bd-11ea-832a-97685243e61c.png)
Let's see how adding pictures work. In short it won't build so can't accept this.